### PR TITLE
Promoted AUTOSCROLL_ACTIVE_OFFSET and AUTOSCROLL_SPEED_RATIO to props

### PR DESCRIPTION
**Pull request by peterkogo**
After a long search for a simple, extensible, reliable, mobile-ready drag-and-drop-list implementation with variable height support - I found this beautiful little project. 

Only thing really preventing this from being 100% extensible and awesome are those hardcoded values mentioned in #68. 

Hopefully I got this right. Current implementations shouldn't break because the previously hardcoded values are now the default values.

### CODE REVIEW

**Comment by tajo** [comment]
@PeterKey  Can you rebase master? It should fix the CI. 

You should also update the documentation.

File: src/List.tsx [hunk 1]
```diff
@@ -315,15 +315,13 @@ class List<Value = string> extends React.Component<IProps<Value>> {
       // autoscrolling for containers with overflow
       if (height + 20 < this.listRef.current!.scrollHeight) {
         let scrollingSpeed = 0;
-        if (clientY - top < AUTOSCROLL_ACTIVE_OFFSET) {
+        if (clientY - top < autoscrollActiveOffset) {
```
Comment (cc-ebay) [inline src/List.tsx:318]: This comment is related to the original logic that this PR does not change. 

If I understand correctly the logic here, this is related to a scrollable container. There is the assumption that `listRef` is it. Assuming that the assumption is right, the math below should not be related to the size of that scrollable container? In particular should not the `autoscrollActiveOffset` be a ratio to the `height` of that scrollable container?

**Review by cc-ebay — COMMENTED** [review]
This is a nice component but I think the autoscrolling logic need some changes.

**Comment by PeterKey** [comment]
@tajo Sure, can do. Will have some time this weekend.

**Comment by PeterKey** [comment]
@cc-ebay In my opinion, the autoscroll behaviour should not change if the scroll container resizes in height. I want to set a fixed pixel value from top/bottom where autoscrolling should commence, so I can optimise it how it feels. This should not change over different screen sizes.

I agree with you however when the scroll containers size gets smaller than 2x the `autoscrollActiveOffset`. Here problems arise and autoscroll behaviour should default to a ratio. 

Now that I think about it, this might also apply if list items are bigger than the scroll container in general: It might make sorting impossible because the list would always scroll. But this is a very weird edge case, that somehow defeats the idea behind a list anyway.

Maybe open a PR or a new issue for this because I want to keep my PR here simple and without breaking changes.

**Pull request closed without merging**